ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/factory/json/ImportFromConfigViewImpl.java

```diff
@@ -50,24 +50,29 @@ public ImportFromConfigViewImpl(
     this.setTitle(locale.importFromConfigurationTitle());
     setWidget(importFromConfigViewBinder.createAndBindUi(this));
 
-    Button btnCancel =
-        createButton(
-            locale.cancelButton(),
-            "import-from-config-btn-cancel",
-            event -> delegate.onCancelClicked());
-    addButtonToFooter(btnCancel);
-
     buttonImport =
         createButton(
             locale.importButton(),
             "import-from-config-btn-import",
             event -> delegate.onImportClicked());
+    buttonImport.addStyleName(resources.windowCss().primaryButton());
```

Comment:
Take a look on other three methods `createButton` in Window class, there is another method which provide _Primary_ style and custom alignment.